compiler/rustc_resolve/src/build_reduced_graph.rs

```diff
@@ -1071,7 +1071,8 @@ impl<'a, 'b, 'tcx> BuildReducedGraphVisitor<'a, 'b, 'tcx> {
             let import = macro_use_import(self, span);
```

Comment:
Could you also make sure that named macro uses (`#[macro_use(foo)]`) on extern crates do not allow using private macros?
(That's the `else` part of this `if`.)

Reply:
I have updated this patch.

Reply:
Ah, great, it is already working correctly.